python/enthic/arbre_compte_de_resultat.py

```diff
@@ -0,0 +1,385 @@
+# -*- coding: utf-8 -*-
+"""
+============================================================
+Tree view of data contained in a french "compte de resultat"
+============================================================
+"""
+
+TREE_VIEW = {
```

Comment:
The keys must be English, to be consistent with the other ontology.

Reply:
Translating technical terms has no use, it's error prone, add complexity and long to do...